Acceleratio.XCellKit/ExcelExport/Chart/Chart.cs

```diff
@@ -0,0 +1,192 @@
+﻿using DocumentFormat.OpenXml;
+using DocumentFormat.OpenXml.Drawing.Charts;
+using DocumentFormat.OpenXml.Packaging;
+using DocumentFormat.OpenXml.Spreadsheet;
+using System.IO;
+using System.Linq;
+using Acceleratio.XCellKit.ExcelExport.Helpers;
+using Chart = DocumentFormat.OpenXml.Drawing.Charts.Chart;
+
+namespace Acceleratio.XCellKit.ExcelExport
+{
+    public class Charts : ReportExport
+    {
+        public Charts(ReportExport export)
+            : base(export)
+        {
+
+        }
+        /// <summary>
+        /// Creates, designes and places chart into a sheet.
+        /// Calls overrideable methods for displaying the chart based on ChartType.
+        /// </summary>
+        public void InsertChartInSpreadsheet(Stream stream)
```

Comment:
the AddChart function should have a parameter of type {some class that is ours}, just as AddTable has SpredsheetTable. SpreddsheetTable provides the abstraction so that the client code(i.e. from SysKit) does not have to deal with OpenXml directly. When i mentioned DrawingPart i did not review the old code so i was one level lower that what is needed, the comment was more along the line that the current implementation where you must modify an existing stream(existing excel file) was not flexible enough. Since i'm on vacation and can't have a direct conversation, speak to Hrvoje one on one. I believe he undestands what i'm getting at and he'll be able to explain better in person than I trough these comments.